web_external/Studies/studyPage.pug

```diff
@@ -28,14 +28,25 @@
       td.isic-study-users-content
         if study.get('users').length
           table(class=study.canAdmin() ? 'table-hover' : undefined)
+            th(colspan=2)
+            th Num. Completed
+            th Percent Completed
             each user in study.users().models
+              - var numCompleted = 'N/A'
+              - var percentCompleted = 'N/A'
```

Comment:
Maybe set `study.get('images').length` to a variable here, since it's used 3 times.